### PR TITLE
Document worker commands in install doc

### PR DESCRIPTION
**Pull request by dralley**
closes #3430
https://pulp.plan.io/issues/3430

### CODE REVIEW

File: docs/installation/instructions.rst [hunk 1]
```diff
@@ -50,6 +50,15 @@ PyPI Installation
 
 7. Go through the :ref:`database-install`, :ref:`broker-install`, and `systemd-setup` sections
 
+.. note::
+
+    In place of using the systemd unit files provided in the `systemd-setup` section, you can run
+    the commands yourself inside of a shell. This is fine for development but not recommended in production.
+
+    $ /path/to/python/bin/celery worker -A pulpcore.tasking.celery_app:celery -n resource_manager@%%h -Q resource_manager -c 1 --events --umask 18
```
Comment (daviddavis) [inline docs/installation/instructions.rst]: Is there a way to escape this section?

Comment (dralley) [reply docs/installation/instructions.rst]: what do you mean exactly?

Comment (daviddavis) [reply docs/installation/instructions.rst]: screen shot 2018-03-14 at 2 56 40 pm

Comment (dralley) [reply docs/installation/instructions.rst]: Updated, we'll see if this fixes it.

**Comment by dralley** [comment]
I don't think github renders the rst properly.  It doesn't render the notes, whereas docs.pulproject.org does.

**Comment by daviddavis** [comment]
Yea, I just use Github as quick way to view the text.

I think if you replace `production.` with `production.::` and remove the backticks, you'll be good.

Also, mind throwing some `$` at the beginning of each of lines. Seems like other places are doing that.

**Pull request merged**